samples/index_tuning_sample/create_vector_embeddings.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import uuid
+
+from langchain.docstore.document import Document
+from langchain_community.document_loaders import CSVLoader
+from langchain_google_vertexai import VertexAIEmbeddings
+
+from langchain_google_alloydb_pg import AlloyDBEngine, AlloyDBVectorStore, Column
+
+EMBEDDING_COUNT = 100
+
+# AlloyDB info
+PROJECT_ID = ""
+REGION = ""  # @param {type:"string"}
+CLUSTER_NAME = ""  # @param {type:"string"}
+INSTANCE_NAME = ""  # @param {type:"string"}
+DATABASE_NAME = ""  # @param {type:"string"}
+USER = ""  # @param {type:"string"}
+PASSWORD = ""  # @param {type:"string"}
+
+vector_table_name = "wine_reviews_vector"
+
+# Dataset
+DATASET_PATH = "wine_reviews_dataset.csv"
+dataset_columns = [
+    "country",
+    "description",
+    "designation",
+    "points",
+    "price",
+    "province",
+    "region_1",
+    "region_2",
+    "taster_name",
+    "taster_twitter_handle",
+    "title",
+    "variety",
+    "winery",
+]
+
+
+async def load_csv_documents(dataset_path=DATASET_PATH):
+    """Loads documents directly from a CSV file using LangChain."""
+
+    loader = CSVLoader(file_path=dataset_path)
+    documents = loader.load()
+
+    documents = [
+        Document(page_content=str(doc.dict()), metadata=doc.metadata)
```

Comment:
Why do we need to reconstruct the Document? loader.load() should give us a list of documents. Unless we are trying to have a specific set of fields in the page content but that should be configured on the loader.